app/services/recaptcha_validator.rb

```diff
@@ -1,14 +1,21 @@
 # frozen_string_literal: true
 
 class RecaptchaValidator
```

Comment:
should we rename to `Form` and move to `app/forms` as well?

Reply:
Yeah, I thought about that. We probably should, wasn't sure if the amount of noise it might add to this pull request might be worth doing it incrementally (i.e. separate pull request), especially since there's subclasses that might also need to be moved.

I did find that we have a couple other examples of form validators in `services`, though I think those probably ought to be moved as well.

Reply:
Some of the `app/validator` objects don't have the same interface as forms, but yes `SamlRequestValidator` seems to implement the `Form` interface